utils/hono-influxdb-connector/src/main/resources/trusted-certs.pem

```diff
@@ -0,0 +1,65 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I removed them from the resources as those demo files should not go with the sources. We can track the respective hono demo certs at the deployment directories